### ISSUE TITLE
Add function/vignette for conditional plotting based on missingness indicator

### ISSUE THREAD

**Issue by hanneoberman** [amices/ggmice#88]
E.g.
```
ggmice(boys, aes(age)) + 
  geom_histogram() + 
  facet_wrap(vars(is.na(gen)))
```

**Comment by hanneoberman** [comment]
Probably nice to add 

```
ggplot2::update_geom_defaults("bar", list(fill = "white"))
```

to `theme_mice()` then

**Comment by hanneoberman** [comment]
Fixed in 556ede906e0a254ef5387d527d620f7ae7962196
